app/modules/feeds-and-posts.js

```diff
@@ -0,0 +1,57 @@
+import { Promise } from 'rsvp';
+
+let fpPromise = null;
+
+/**
+ * Gets the Feeds & Posts ES Module, by first appending the F&P library to the page
+ *
+ * @returns {Promise} Promise of module
+ */
+function getModule() {
+	if (fpPromise) {
+		return fpPromise;
+	}
+
+	fpPromise = new Promise((resolve, reject) => {
+		const fpScript = document.createElement('script');
+		fpScript.onload = () => {
+			resolve(window.fp);
+		};
+		fpScript.onerror = () => {
+			reject();
+		};
+		fpScript.src = '/feeds-and-posts/public/dist/lib.min.js';
+		document.querySelector('head').appendChild(fpScript);
+	});
+
+	return fpPromise;
+}
+
+/**
+ * Loads the feed onto an article page
+ *
+ * @param {Module} module
+ */
+function loadFeed(module) {
```

Comment:
this could be rewritten to a pattern that is closer to React way of thinking (if I get it correctly) and it could use a component wrapper to handle the logic
here is a quick example that I did
https://ember-twiddle.com/af5a81d96e4bc50d98f30c74542dbadc?openFiles=templates.components.my-component.hbs%2Ctemplates.components.my-component.hbs

Reply:
then loading of module could also happen in this component
and we could enable/disable feeds by rendering or not the compenent